backbone.js

```diff
@@ -149,10 +146,20 @@
     }
 
     // Bind callbacks on obj, and keep track of them on listening.
-    internalOn(obj, name, callback, this, listening);
+    var error = tryCatchOn(obj, name, callback, this);
+    listening = void 0;
```

Comment:
It's [now](https://github.com/jashkenas/backbone/pull/3615/files#diff-0d56d0d310de7ff18b3cef9c2f8f75dcR91) a private global.